package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "document-builder-declarations",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "build": "pnpm make build",
+    "make": "node ./makefile.js"
+  },
+  "devDependencies": {
+    "axios": "^1.7.5",
+    "sade": "^1.8.1"
+  },
+  "packageManager": "pnpm@9.1.3",
+  "engines": {
+    "node": "21.2.0"
+  }
```

Comment:
Remove the `packageManager` proeprty and replace node version with `^22` from the `.mice.toml`.